packages/backend/src/utils.ts

```diff
@@ -48,6 +48,16 @@ export const excludeReposByName = <T extends Repository>(repos: T[], excludedRep
     });
 }
 
+export const includeReposByName = <T extends Repository>(repos: T[], includedRepoNames: string[], logger?: Logger) => {
+    return repos.filter((repo) => {
+        if (micromatch.isMatch(repo.name, includedRepoNames)) {
+            logger?.debug(`Including repo ${repo.id}. Reason: repos does not contain ${repo.name}`);
```

Comment:
micro nit: Should be "repos contains ${repo.name}"

Reply:
fixed